src/Buerokratt.Common/Dmr/DmrService.cs

```diff
@@ -32,23 +39,22 @@ public override async Task ProcessRequestAsync(DmrRequest payload)
             try
             {
                 // Setup message
-                using var requestMessage = CreateRequestMessage(payload);
+                using var requestMessage = await CreateRequestMessage(payload);
 
                 // Send request
                 var response = await HttpClient.SendAsync(requestMessage).ConfigureAwait(false);
-
-                if (!response.IsSuccessStatusCode)
-                {
-                    var errorReason = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
I think we might be losing the response in the body (the `errorReason`) that the DMR is potentially returning to us, no?

Reply:
You know - I think you're absolutely right.  I'd convinced myself EnsureSuccessStatusCode was cleverer than it was... I've put this back,